packages/grafana-data/src/types/pluginExtensions.ts

```diff
@@ -208,6 +208,11 @@ export enum PluginExtensionPoints {
   ExtensionSidebar = 'grafana/extension-sidebar/v0-alpha',
 }
 
+// Extension Points available in plugins
+export enum PluginExtensionPointsInPlugins {
+  IrmAlertRuleHistory = 'grafana-irm-app/alert-rule-history/v1',
+}
+
```

Comment:
@jackw do you know the status on the `@grafana/plugin-types` package? Could this live in that package and we take a dependency on it from Grafana?

Reply:
Yea, we have https://github.com/grafana/plugin-extension-types which can be used to share types between plugins, and a few plugins are already using it :)